ClientSide(FrontEnd)/clay_/src/components/Images.js

```diff
@@ -0,0 +1,106 @@
+import './Images.css';
+ import img1 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/components/home.jpg';
+ import img2 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/components/WhatsApp Image 2023-05-30 at 5.09.49 PM(2).jpeg';
+ import img3 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/WhatsApp Image 2023-05-30 at 5.09.36 PM.jpeg';
+ import img4 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/WhatsApp Image 2023-05-30 at 5.09.46 PM(1).jpeg';
+ import img5 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/components/WhatsApp Image 2023-05-30 at 5.09.33 PM(2).jpeg';
+ import img6 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/image.jpg';
+ import img7 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-05 19-53-44.png';
+ import img8 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-05 19-53-25.png';
+ import img9 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/download.png';
+ import img10 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/dolor.png';
+ import img11 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/bell2.jpg';
+ import img12 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/bell2.jpg';
+ import img13 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/OTP-METAL25.webp';
+ import img14 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-06 00-21-36.png'
+function Images(){
+
+    return(
+        <div>
+            <div>
+                <img className="img1" src={img1} alt=""></img>
+                <div className="nav">
+                    <div>Home</div>
+                    <div>Category</div>
+                    <div>About Us</div>
+                    <button className="btn">Register</button>
+                    <button className="btn">Add to cart</button>
+                </div>
+            
+            </div>
+            <div className="box">
+                <div className="second">
+                    <div className="image">
+                        <img className="img2" src={img2} alt=""></img>
```

Comment:
please give alternative name instead of empty string in the entire document

Reply:
i gave alternative name instead of empty string